src/constants/index.ts

```diff
@@ -23,6 +23,7 @@ export const GameScene1 = "GameScene1";
 export const EndScene1 = "EndScene1";
 export const FONT_BASE_PATH = './assets/fonts/';
 export const feedbackTextDefault = "Atma-SemiBold";
+export const titleTextDefaultFont = "Atma-SemiBold";
```

Comment:
You keep creating functions that will handle the font change but you are not using those.

Please utilize the functions you have created instead of declaring them as a constant.

Reply:
I agree with @bernhardccodev on this
@woodsjohn-codev please consider using the same.

Reply:
applied changes here. thanks guys